Marlin/ultralcd_impl_HD44780.h

```diff
@@ -795,7 +795,9 @@ static void lcd_implementation_status_screen() {
 
   #endif // FILAMENT_LCD_DISPLAY && SDSUPPORT
 
-  lcd_print(lcd_status_message);
+  char status_message[LCD_WIDTH + 1];
+  strlcpy(status_message, lcd_status_message, sizeof(status_message));
```

Comment:
Currently clipping, but in future we will likely add a feature that scrolls an over-long status message.